.github/workflows/ior.yml

```diff
@@ -0,0 +1,87 @@
+name: ior+mobject@develop
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+    - 'doc/**'
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install APT Dependencies
+      run: |
+        sudo apt update
+        sudo apt-get install -y autoconf
+        sudo apt-get install -y automake
+        sudo apt-get install -y cmake
+        sudo apt-get install -y libtool
+        sudo apt-get install -y libtool-bin
+        sudo apt-get install -y mpich
+        sudo apt-get install -y zlib1g-dev
+        sudo apt-get install -y diffutils
+        sudo apt-get install -y libdb-dev
+        sudo apt-get install -y libedit-dev
+        sudo apt-get install -y libncurses6
+        sudo apt-get install -y libxml2-dev
+        sudo apt-get install -y openssl
+    - name: Get Sources
+      uses: actions/checkout@v3
+    - name: Checkout Spack
+      uses: actions/checkout@v3
+      with:
+        repository: spack/spack
+        path: ./spack
+    - name: Checkout mochi-spack-packages
+      uses: actions/checkout@v3
+      with:
+        repository: mochi-hpc/mochi-spack-packages
+        path: ./mochi-spack-packages
+    - name: Run spack install ior
+      run: |
+        . ./spack/share/spack/setup-env.sh
+        spack compiler find
+        spack compilers
+        spack external find autoconf
+        spack external find automake
+        spack external find berkeley-db
+        spack external find boost
+        spack external find cmake
+        spack external find diffutils
+        spack external find libedit
+        spack external find libtool
+        spack external find libiconv
+        spack external find libxml2
+        spack external find m4
+        spack external find mpich
+        spack external find ncurses
+        spack external find openssl
+        spack external find perl
+        spack external find pkgconf
+        spack external find zlib
+        spack external find xz
+        spack repo add ./mochi-spack-packages
+        spack install mobject@develop+bedrock
+        spack load mobject
+        spack install ior@develop+mobject
```

Comment:
I think you could do this with one spack install line since the `ior+mobject` variant will pull in bedrock already.  It also looks like the build is taking over 20 minutes; I suspec that a lot of this is because it is building boost from scratch.

Maybe try `spack install ior@develop+mobject ^mobject@develop+bedrock ^mercury@develop~boostsys`?  I think that will do it all in one spack install command, and will tell the Mercury package to use it's own internal boost headers rather than building all of boost.

Reply:
@carns , I get the following error if I try one liner install command for Spack:

```
==> Error: No version for 'mercury' satisfies '@develop-multi_recv' and '@master'
```